src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py

```diff
@@ -470,15 +471,17 @@ def upload_plugins(s3_client, plugin_files: Dict[str, str]) -> Optional[Response
         with open(os.path.join("/tmp/model_plugins/", path), "w") as f:
             f.write(contents)
 
-    provision_schemas(
+    th = threading.Thread(target=provision_schemas, args=(
         GraphClient(),
         raw_schemas,
-    )
+    ))
+    th.start()
 
     for path, file in plugin_files.items():
         upload_resp = upload_plugin(s3_client, path, file)
         if upload_resp:
             return upload_resp
+    th.join()
```

Comment:
does this thread get auto joined if we ever `return upload_resp` or does it just spin forever

Reply:
it'll die eventually, but yeah probably worth a finally or something